docs/source/notebooks/temp-sense-gen/temp_sense_genCollab.ipynb

```diff
@@ -122,14 +122,16 @@
     "import pathlib\n",
     "import sys\n",
     "\n",
-    "!apt-get install -y ruby-full time build-essential\n",
-    "!apt install -f libqt4-designer libqt4-xml libqt4-sql libqt4-network libqtcore4 libqtgui4\n",
+    "!apt install -y time build-essential\n",
+    "!apt install qt5-default qttools5-dev libqt5xmlpatterns5-dev qtmultimedia5-dev libqt5multimediawidgets5 libqt5svg5-dev -y\n",
+    "!apt install ruby ruby-dev libz-dev python3-dev -y\n",
+    "!wget https://www.klayout.org/downloads/Ubuntu-20/klayout_0.28.3-1_amd64.deb\n",
+    "!dpkg -i klayout_0.28.3-1_amd64.deb\n",
     "!curl -Ls https://micro.mamba.pm/api/micromamba/linux-64/latest | tar -xvj bin/micromamba\n",
     "conda_prefix_path = pathlib.Path('conda-env')\n",
     "CONDA_PREFIX = str(conda_prefix_path.resolve())\n",
     "%env CONDA_PREFIX={CONDA_PREFIX}\n",
     "!bin/micromamba create --yes --prefix $CONDA_PREFIX\n",
-    "!echo 'python ==3.7*' >> {CONDA_PREFIX}/conda-meta/pinned\n",
```

Comment:
it'd say that it is necessary to pin the python version in the environment because conda-eda only support 3.7 for now:
https://github.com/hdl/conda-eda/issues/260

Reply:
oay, so that means 3.7 is fine for now? How do we make sure this is automatically updated?

Reply:
When I executed the notebook, even if I pin python 3.7, it still used python3.8 directory for some reason. But, when I remove the pin, the notebook runs smoothly. I'll check the notebook again.

Reply:
I think the problem is not on this "echo python ==3.7". The problem is some of the PATH information at the end of the code cell. I currently testing the line that caused the problem specifically.